pagopa/io-app
# fix(IT Wallet): [SIW-1526] Wrong date claim with different timezone

## Short description
This PR fixes an issues which causes the date claims to consider the timezone of the device.

## List of changes proposed in this pull request
- Add the timezone to the date object which might be positive or negative, based on the timezone, to bring it back to UTC format. 

## How to test
- Activate the wallet;
- Obtain a MDL;
- Open the credential detail to check for date claims which should be same as your physical document;
- Change the timezone of the device or the date and shift it by a day. The date claims should stay the same in the credential details.
